hack/verify-generated-protobuf.sh

```diff
@@ -23,7 +23,7 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 kube::golang::setup_env
 
-APIROOTS=${APIROOTS:-pkg/api pkg/apis pkg/watch staging/src/k8s.io/apimachinery/pkg/api staging/src/k8s.io/apimachinery/pkg/apis staging/src/k8s.io/apiserver/pkg staging/src/k8s.io/api staging/src/k8s.io/metrics/pkg/apis}
+APIROOTS=${APIROOTS:-pkg/api pkg/apis pkg/watch staging/src/k8s.io/apimachinery/pkg/api staging/src/k8s.io/apimachinery/pkg/apis staging/src/k8s.io/apiserver/pkg staging/src/k8s.io/api staging/src/k8s.io/metrics/pkg/apis staging/src/k8s.io/sample-apiserver/pkg/apis}
```

Comment:
Am not convinced of just adding this here. We have sample-apiserver as an example repository (https://github.com/kubernetes/sample-apiserver). It is supposed to work outside of the main k/k repo. In other words, we need proto support in https://github.com/kubernetes/code-generator/blob/master/generate-internal-groups.sh which is called from sample-apiserver.

Reply:
Gotcha, I will look into doing that instead. Just for documentation, my line of thought was that:

*From https://github.com/kubernetes/sample-apiserver: "Where does it come from?
sample-apiserver is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/sample-apiserver. Code changes are made in that location, merged into k8s.io/kubernetes and later synced here."

* kubernetes/hack/update-generated-protobuf.sh would update/gen protobuf in kubernetes/staging/src/k8s.io/sample-apiserver which gets synced to the sample-apiserver repo.

Reply:
Would it be sufficient for this PR to add the update/verify protobuf scripts to the sample-apiserver repo, then file a new issue to dissolve the protobuf update scripts into the code-generator repo and update/verify codegen scripts?

Reply:
Just do both here I would say.